safe_transaction_service/history/services/safe_service.py

```diff
@@ -220,10 +224,10 @@ def _get_next_internal_tx(self, internal_tx: InternalTx) -> Optional[InternalTx]
     def _get_parent_internal_tx(self, internal_tx: InternalTx) -> InternalTx:
         if parent_trace := internal_tx.get_parent():
             return parent_trace
-        if not self.tracing_enabled:
```

Comment:
Was this field deprecated?

Reply:
It's not required anymore, checking `if not self.ethereum_tracing_client` is enough